### ISSUE TITLE
Jetty 12 documentation zip is practically empty during release

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#10449]
**Jetty version(s)**
12.0.1

**Jetty Environment**
n/a

**Java version/vendor**
n/a

**OS type/version**
n/a

**Description**
The latest release 12.0.1 has a practically empty documentation zip.

https://repo1.maven.org/maven2/org/eclipse/jetty/documentation/jetty-documentation/12.0.1/jetty-documentation-12.0.1-html.zip

We have to fix this for the next 12.x release.

**Comment by Quantum2Bits** [comment]
The API documentation page shows only "Documentation :: Guides 12.0.1 API" on https://eclipse.dev/jetty/javadoc/jetty-12/index.html?overview-summary.html. There are only packages org.eclipse.jetty.docs.programming.* there. All other packages are absent.

**Comment by joakime** [comment]
Yeah, that's not intended.
Thankfully we can fix this before 12.0.2 (next month).

**Comment by sbordet** [comment]
@joakime we don't want to show any javadocs for those -- they are the examples used in the docs.

**Comment by olamy** [comment]
@joakime did you use the script?
if not remember to add `-Dasciidoctor.skip=false` to the command if you use `-DskipTests`
Because of this https://github.com/eclipse/jetty.project/blob/896a301fc10b9e0ce88242bc5f232ce24c9fb4d5/documentation/jetty-documentation/pom.xml#L16

Such `mvn clean deploy -Peclipse-release -Dmaven.build.cache.enabled=false -DskipTests -Dasciidoctor.skip=false` 

As far I can see it's part of the script https://github.com/eclipse/jetty.project/blob/896a301fc10b9e0ce88242bc5f232ce24c9fb4d5/build/scripts/release-jetty.sh#L91https://github.com/eclipse/jetty.project/blob/896a301fc10b9e0ce88242bc5f232ce24c9fb4d5/build/scripts/release-jetty.sh#L91

**Comment by olamy** [comment]
solution could be adding in profile `eclipse-release` 
```
<asciidoctor.skip>false</asciidoctor.skip>
```

**Comment by olamy** [comment]
@joakime PR https://github.com/eclipse/jetty.project/pull/10481

**Comment by olamy** [comment]
fixed by #10481

**Comment by joakime** [comment]
Opened updated PR #10532 to address more things with jetty-documentation

**Comment by joakime** [comment]
Merged PR #10549.

I tested the current `jetty-12.0.x` HEAD (commit 9ce414125de8d80d7c39cbda67fd600098f3bf6e) with a simulated deploy using the release profile.

``` shell
# Tip: start with a clean/empty $HOME/tmp/alt-repo/ directory
$ mvn clean deploy -Pci -Peclipse-release -Dmaven.build.cache.enabled=false -DskipTests -DaltDeploymentRepository=oss.sonatype.org::default::file:$HOME/tmp/alt-repo/
```

The deployed jetty-documentation artifacts are looking good ...

``` shell
[~/tmp/alt-repo]$ ls -la org/eclipse/jetty/documentation/jetty-documentation/12.0.2-SNAPSHOT/
total 17872
drwxrwxr-x 2 joakim joakim     4096 Sep 28 10:29 .
drwxrwxr-x 3 joakim joakim     4096 Sep 28 10:29 ..
-rw-rw-r-- 1 joakim joakim  4242911 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.sha1
-rw-rw-r-- 1 joakim joakim 13983283 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.sha1
-rw-rw-r-- 1 joakim joakim    14474 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.sha1
-rw-rw-r-- 1 joakim joakim     1669 Sep 28 10:29 maven-metadata.xml
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 maven-metadata.xml.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 maven-metadata.xml.sha1
```

Closing Issue.
